pt-BR/mail/chrome/messenger/localMsgs.properties

```diff
@@ -243,26 +243,26 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Please enter a new password for use
 
 ## @name POP3_AUTH_INTERNAL_ERROR
 ## @loc None
-4047=Internal state error during POP3 server authentication. This is an internal, unexpected error in the application, please report it as bug.
+4047=Erro interno de estado durante a autenticação no servidor POP3. Este é um erro interno e inesperado do aplicativo. Por favor, reporte como uma falha.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 ## @loc None
-4048=This POP3 server does not seem to support encrypted passwords. If you just set up the account, please try changing to 'Password, transmitted insecurely' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, this is a common scenario how someone could steal your password.
+4048=Este servidor POP3 parece não suportar senhas criptografadas. Se você acabou de configurar esta conta, por favor, tente alterar o 'Método de autenticação' para 'Senha enviada de forma insegura' em 'Configurações de conta' / 'Configurações do servidor'. Se antes funcionava e agora parou de funcionar de repente, é um possível cenário de roubo.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 ## @loc None
-4049=This POP3 server does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+4049=Este servidor POP3 parece não suportar senhas criptografadas. Se você acabou de configurar esta conta, por favor, tente alterar o 'Método de autenticação' para 'Senha normal' em 'Configurações de conta' / 'Configurações do servidor'. Se antes funcionava e agora parou de funcionar de repente, por favor, entre em contato com o seu administrador ou provedor de e-mails.
 
 ## @name POP3_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 ## @loc None
-4050=This POP3 server does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+4050=Este servidor POP3 não permite senhas em texto claro. Por favor, tente alterar o 'Método de autenticação' para 'Senha criptografada' em 'Configurações de conta' / 'Configurações do servidor'.
```

Comment:
CORRECTIONS NEEDED it should be 4050=Este servidor POP3 não permite senhas como texto não criptografado. Por favor, tente alterar o 'Método de autenticação' para 'Senha criptografada' em 'Configurações de conta' / 'Configurações do servidor'.